src/arkode/arkode_lsrkstep.c

```diff
@@ -2087,15 +2087,18 @@ void lsrkStep_PrintMem(ARKodeMem ark_mem, FILE* outfile)
             step_mem->dom_eig_num_evals);
 
     /* output sunrealtype quantities */
-    fprintf(outfile, "LSRKStep: dom_eig               = %" RSYM " %+" RSYM "i\n",
+    // TODO(SRB): temporary fix for complex numbers
+    fprintf(outfile,
+            "LSRKStep: dom_eig               = " SUN_FORMAT_G
+            " + (" SUN_FORMAT_G ")i\n",
```

Comment:
Are the parenthesis necessary?

Reply:
I see, but it's still a little awkward. Having a `SUN_FORMAT` macro that always includes the sign might be nicer. Maybe something like,
```
#define SUN_FORMAT_SG "%+." SUN_STRING(FLT_DIG) "g"
```